cypress/e2e/models/migration/dynamic-report/issues/issues.ts

```diff
@@ -84,7 +84,7 @@ export class Issues {
             .within(() => {
                 validateTextPresence(issueColumns.issue, issue.name);
                 validateTextPresence(issueColumns.category, issue.category);
-                validateTextPresence(issueColumns.source, issue.source);
+                validateTextPresence(issueColumns.source, issue.source[0]);
```

Comment:
If source is going to be an array now, it should be renamed to `sources`

Reply:
Done